src/components/ConfigFormModal.tsx

```diff
@@ -296,6 +311,15 @@ export const ConfigFormDrawer = forwardRef(({ opened, onClose }: { opened: boole
                     })}
                   />
 
+                  <NumberInput
+                    label={t('pprofPort')}
```

Comment:
Is this required field from a user's perspective?

Reply:
pprofPort doesn't seem to work.

Reply:
> pprofPort doesn't seem to work.

It should not be marked as withAsterisk then

Reply:
I set up pprofPort on daed, but the corresponding port didn't start. I don't know why this happened, so I can only temporarily hide it from the interface.